### PR TITLE
completed 186 issue leave My account and Blue Delta Settings router

### PR DESCRIPTION
**Pull request by chris-saburo**

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -4,12 +4,58 @@
       <div>
         <BaseAppBarHeader title="Settings" :to-link="'/'" />
         <div class="container px-0 md:px-6">
+          <div class="flex items-strech items-center border-b-1 border-divider">
+            <router-link
+              :to="{ name: '' }"
+              class="link-fordward block flex-grow justify-between flex h-full items-center cursor-pointer select-none px-4 pr-6 py-5"
+            >
+              <span class="mr-2 tg-body-mobile">My Account</span>
+              <svg
```
Comment (atharva3010) [inline src/views/Settings.vue:13]: Please save the svg in assets folder and then import it as a component, that is how we use svgs in our codebase.

File: src/views/Settings.vue [hunk 2]
```diff
@@ -4,12 +4,58 @@
...
+              </svg>
+            </router-link>
+          </div>
+          <div class="flex items-strech items-center border-b-1 border-divider">
+            <router-link
+              :to="{ name: '' }"
+              class="link-fordward block flex-grow justify-between flex h-full items-center cursor-pointer select-none px-4 pr-6 py-5"
+            >
+              <span class="mr-2 tg-body-mobile">Blue Delta Settings</span>
+              <svg
```
Comment (atharva3010) [inline src/views/Settings.vue:36]: here too

File: src/views/Settings.vue [hunk 3]
```diff
@@ -4,12 +4,58 @@
...
+            </router-link>
+          </div>
           <div class="flex items-strech items-center border-b-1 border-divider">
             <router-link
               :to="{ name: 'EmailLists' }"
               class="link-fordward block flex-grow justify-between flex h-full items-center cursor-pointer select-none px-4 pr-6 py-5"
             >
-              <span class="mr-2 tg-body-mobile">Email Settings</span>
+              <span class="mr-2 tg-body-mobile">Distribution Groups</span>
               <svg
```
Comment (atharva3010) [inline src/views/Settings.vue:59]: here too

Comment (chris-saburo) [reply src/views/Settings.vue:59]: All fixed.

**Comment by atharva3010** [comment]
Looks good

**Pull request merged**